src/Agent.Worker/Release/Artifacts/JenkinsArtifact.cs

```diff
@@ -118,10 +118,10 @@ public class JenkinsArtifact : AgentService, IArtifactExtension
                         else if (startJobId > endJobId)
                         {
                             context.Output(StringUtil.Loc("JenkinsRollbackDeployment"));
```

Comment:
add the start & end job id to the output string. see how it is done for DownloadingJenkinsCommitsBetween